Formula/lima.rb

```diff
@@ -20,7 +20,7 @@ class Lima < Formula
   depends_on "qemu"
 
   def install
-    system "make", "VERSION=#{version}", "clean", "binaries"
+    system "make", "VERSION=#{version}", "clean", "all"
```

Comment:
This changes is for installing manpages
https://github.com/lima-vm/lima/blob/v0.16.0/Makefile#L35